src/components/Popover.svelte

```diff
@@ -10,7 +10,7 @@
 <script lang="ts">
   export let disabled = false;
 
-  let expanded = false;
+  export let expanded = false;
```

Comment:
I think passing expanded to the slots and [binding it with `let:expanded`](https://svelte.dev/docs/special-elements#slot-slot-key-value) is the more idiomatic approach

Reply:
Thanks for the reminder, this is so much better! 😌 
cdf3ea41b65a69bcaa1eeec982e373b4661cd206